frontend/src/containers/NotificationModal.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div class="scroll">
+    <CNav fill variant="pills" class="mb-3">
+      <CNavItem :active="tab == 0" @click.native="setTab(0)">
+        All Notifications ({{notifications.length}})
+      </CNavItem>
+      <CNavItem :active="tab == 1" @click.native="setTab(1)">
+        Unread Notifications ({{unReadNotif.length}})
+      </CNavItem>
+    </CNav>
+    <hr />
+    <CCard v-if="tab == 0">
+          <CCardBody v-for="notif in notifications"
+          :key="notif._id"
+          class='cursor-pointer'
+          @click="redirectAndMarkRead(notif, true)">
+              <div class="message">
+                  <div>
+                      <small class="blueTxt">{{notif.user}}</small>
+                      <small class="float-right mt-1 blueTxt">{{notif.timestamp}}</small>
+                  </div>
+                  <div class="text-truncate font-weight-bold">{{notif.title}}</div>
+                  <div class="small text-muted text-truncate">{{notif.description}}</div>
+              </div>
+          </CCardBody>
+    </CCard>
+    <CCard v-if="tab == 1">
+        <CCardBody v-for="notif in unReadNotif"
+        :key="notif._id"
+        class='cursor-pointer'
+        @click="redirectAndMarkRead(notif, false)">
+            <div class="message">
+                <div>
+                    <small class="blueTxt">{{notif.user}}</small>
+                    <small class="float-right mt-1 blueTxt">{{notif.timestamp}}</small>
+                </div>
+                <div class="text-truncate font-weight-bold">{{notif.title}}</div>
+                <div class="small text-muted text-truncate">{{notif.description}}</div>
+            </div>
+        </CCardBody>
+    </CCard>
+
+  </div>
+</template>
+
+<script>
+const axios = require("axios");
+
+export default {
+  props: ["setNotifModal"],
+  name: "NotifModal",
+  data() {
+    return {
+      tab: 0,
+      notifications: [],
+      unReadNotif: [],
+      subscribedEvents: null,
+    };
+  },
+  methods: {
+    setTab(tab) {
+      this.tab = tab;
+    },
+    changeCursor() {
```

Comment:
I don’t think this method is used anywhere. If not it should be removed

Reply:
You're right. I was testing something and forgot to delete it. It has been addressed in this [commit](https://github.com/purplesmurf1998/Autotracks/pull/89/commits/0eff0dd34a0c31fe68b66ebdd50fba5193658398)